examples/bert/README.md

```diff
@@ -178,3 +179,24 @@ tensorboard --logdir runs/
 ```
 
 ![Visualizing loss/accuarcy on Tensorboard](tbx.png)
+
+## Hyperparameter tuning with Executor
+
+To run this example, please install `hyperopt` by issuing the following command
+
+```commandline
+pip install hyperopt
+```
+
+`bert_with_tpe.py` shows an example of how to tune hyperparameters with Executor using `hyperopt`. 
+To run this example, run the following command
+
+```commandline
+python bert_with_tpe.py
+```
+
+In this simple example, the hyperparameters to be tuned are provided as a `dict` in
+`bert_tpe_config_classifier.py` which are fed into `objective_func()` . We use `TPE` algorithm for
```

Comment:
Add the full name of TPE. E.g., We use `TPE` (xxxx) algorithm ...

Reply:
Added in e175fa6